src/test/java/seedu/address/logic/commands/ConfirmationAcceptCommandTest.java

```diff
@@ -0,0 +1,114 @@
+
+        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS, editedPerson);
+
+        Model expectedModel = new ModelManager(new AddressBook(model.getAddressBook()), new UserPrefs());
+        expectedModel.setPerson(model.getFilteredPersonList().get(0), editedPerson);
+
+        ConfirmationAcceptCommand confirmationAcceptCommand = new ConfirmationAcceptCommand(editCommand);
+        assertCommandSuccess(confirmationAcceptCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_invalidEditCommand_failure() {
+        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredPersonList().size() + 1);
+        EditCommand.EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder()
+                .withName(VALID_NAME_BOB).build();
+        EditCommand editCommand = new EditCommand(outOfBoundIndex, descriptor);
+
+        ConfirmationAcceptCommand confirmationAcceptCommand = new ConfirmationAcceptCommand(editCommand);
+        assertCommandFailure(confirmationAcceptCommand, model, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void execute_acceptClearCommand_success() {
+        ClearCommand clearCommand = new ClearCommand();
+        Model expectedModel = new ModelManager();
+
+        ConfirmationAcceptCommand confirmationAcceptCommand = new ConfirmationAcceptCommand(clearCommand);
+        assertCommandSuccess(confirmationAcceptCommand, model, ClearCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+
+    @Test
+    public void equals() {
+        DeleteCommand firstDeleteCommand = new DeleteCommand(INDEX_FIRST_PERSON);
+        DeleteCommand secondDeleteCommand = new DeleteCommand(INDEX_SECOND_PERSON);
+
+        Person editedPerson = new PersonBuilder().build();
+        EditCommand.EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(editedPerson).build();
+        EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, descriptor);
+
+        ClearCommand clearCommand = new ClearCommand();
+
+        ConfirmationAcceptCommand confirmationAcceptDeleteCommand = new ConfirmationAcceptCommand(firstDeleteCommand);
+        ConfirmationAcceptCommand confirmationAcceptEditCommand = new ConfirmationAcceptCommand(editCommand);
+        ConfirmationAcceptCommand confirmationAcceptClearCommand = new ConfirmationAcceptCommand(clearCommand);
+
+        assertTrue(confirmationAcceptDeleteCommand.equals(confirmationAcceptDeleteCommand));
+        assertTrue(confirmationAcceptEditCommand.equals(confirmationAcceptEditCommand));
+        assertTrue(confirmationAcceptClearCommand.equals(confirmationAcceptClearCommand));
+
+        assertFalse(confirmationAcceptDeleteCommand.equals(confirmationAcceptClearCommand));
+        assertFalse(confirmationAcceptDeleteCommand.equals(confirmationAcceptEditCommand));
+        assertFalse(confirmationAcceptEditCommand.equals(confirmationAcceptClearCommand));
```

Comment:
I noticed a similar issue previously. Maybe more comments explaining each test case could be added?

Reply:
Thank you for the comment, I have resolved it!